cmd/stack_validate.go

```diff
@@ -148,7 +153,7 @@ Runs the following validation tests against the stack and its templates:
 				rootConfig.Info.Log("Created project dir: " + projectDir)
 
 				// init
-				err = TestInit(rootConfig.LoggingConfig, "dev.local/"+stackName, templates[i], projectDir)
+				err = TestInit(rootConfig.LoggingConfig, imageRegistry+imageNamespace+stackName, templates[i], projectDir)
```

Comment:
Need a "/"s between the parts. Since this is used multiple times, create a variable with the fully qualified image once to use it in all the commands.

Reply:
Just realised it doesn't make sense to have registry and namespace here...I will remove it altogether